core/src/components/back-button/back-button.tsx

```diff
@@ -24,7 +24,7 @@ import { createColorClasses, hostContext, openURL } from '../../utils/theme';
   shadow: true
 })
 export class BackButton implements ComponentInterface, ButtonInterface {
-  private inheritedAttributes: { [k: string]: any } = {};
+  private inheritedAttributes: Attributes = {};
```

Comment:
There are several other components that use `{ [k: string]: any }`. Can we either update them all now or update them all in a separate PR?

Reply:
Updated all the implementations surrounding inheriting attributes with the new type.